File: CSharp/Contests2025/ABC407/C.cs
class C
{
	static void Main() => Console.WriteLine(Solve());
	static object Solve()
	{
		var s = Console.ReadLine();
		var n = s.Length;

		s += "0";
		return n + Enumerable.Range(0, n).Sum(i => (s[i] - s[i + 1] + 10) % 10);
	}
}

File: Java/Contests2025/ABC407/src/C.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.stream.IntStream;

public class C {
	static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));

	public static void main(String[] args) throws IOException {
		System.out.println(solve());
	}

	static Object solve() throws IOException {
		var s = in.readLine() + "0";
		var n = s.length() - 1;

		return n + IntStream.range(0, n).map(i -> (s.charAt(i) - s.charAt(i + 1) + 10) % 10).sum();
	}
}
